app/controllers/api/hud_filters_controller.rb

```diff
@@ -26,12 +26,12 @@ def index
         projects.pluck(
           :id,
           :ProjectName,
+          :confidential,
           :computed_project_type,
           o_t[:OrganizationName],
           o_t[:id],
-        ).each do |id, p_name, type, o_name, o_id|
-          name = GrdaWarehouse::Hud::Project.confidentialize(name: p_name)
-          name = p_name if can_view_confidential_enrollment_details?
+        ).each do |id, p_name, p_confidential, type, o_name, o_id|
+          name = GrdaWarehouse::Hud::Project.confidentialize_name(current_user, p_name, p_confidential)
```

Comment:
Is there a vector here for injecting `confidential=false` in the url to get the list of projects to include the confidential projects?  Not sure that matters, this is just a list of projects, no enrollments attached, but wanted to check.

Reply:
No, because the visibility is based on `current_user` and the `confidential` flag is pulled directly from the project table. It behaves the same way regardless of what params are passed to the endpoint